src/components/ui/QuestionCard.tsx

```diff
@@ -1,21 +1,21 @@
-import React, { useState } from "react"
-import Loader from "./Loader"
-import Delete from "../../assets/DeleteIcon"
+import { useState } from "react";
+import Loader from "./Loader";
+import Delete from "../../assets/DeleteIcon";
 import {
   useDeleteQuestionMutation,
   useEditQuestionMutation,
-} from "../../features/user/backendApi"
-import SuccessCheckMark from "../../assets/SuccessCheckMarkIcon"
-import { useForm } from "react-hook-form"
-import AddIcon from "../../assets/AddIcon"
-import RemoveIcon from "../../assets/RemoveIcon"
-import Reset from "../../assets/ResetIcon"
-import DeleteModal from "../modals/DeleteModal"
-import { Cookie, QuestionType } from "../../utils/types"
-import { useCookies } from "react-cookie"
+} from "../../features/user/backendApi";
+import SuccessCheckMark from "../../assets/SuccessCheckMarkIcon";
+import { useForm } from "react-hook-form";
+import AddIcon from "../../assets/AddIcon";
+import RemoveIcon from "../../assets/RemoveIcon";
+import Reset from "../../assets/ResetIcon";
+import DeleteModal from "../modals/DeleteModal";
+import { Cookie, QuestionType } from "../../utils/types";
+import { useCookies } from "react-cookie";
 
-const QuestionCard = ({ question }: any) => {
-  const { prompt, type, options, _id } = question
+const QuestionCard = ({ question, readonly }: { question: any; readonly: boolean }) => {
+  const { prompt, type, options, _id } = question;
```

Comment:
```suggestion
const QuestionCard = ({ question, readonly }: { question: TemplateQuestion; readonly: boolean }) => {
  const { prompt, type, options, _id } = question;
```

I think here we can properly annotate the question type instead of using `any`